tests/python_functional/src/setup/testcase.py

```diff
@@ -86,10 +86,8 @@ def new_syslog_ng(self, instance_name="server"):
     def new_syslog_ng_ctl(self, syslog_ng):
         return SyslogNgCtl(self.__logger_factory, syslog_ng.instance_paths)
 
-    def new_config(self, version=None):
-        if not version:
-            version = self.__testcase_context.getfixturevalue("version")
-        return SyslogNgConfig(self.__logger_factory, self.__testcase_parameters.get_working_dir(), version)
+    def new_config(self):
```

Comment:
The original signature would worth maintaining, so that a test could override config version.

Reply:
I have removed this commit